addons/web_tour/static/src/js/tip.js

```diff
@@ -155,25 +250,49 @@ var Tip = Widget.extend({
         if (rtlMap[appendAt] && _t.database.parameters.direction === 'rtl') {
             appendAt = rtlMap[appendAt];
         }
-        this.$el.position({
-            my: this._get_spaced_inverted_position(appendAt),
-            at: appendAt,
-            of: this.$anchor,
-            collision: "none",
-        });
+
+        // Get the correct tip's position depending of the tip's state
+        let $parent = this.$ideal_location;
+        if ($parent.is('html,body') && this.viewPortState !== "in") {
+            this.$el.css({position: 'fixed'});
+        }
+        const paddingTop = parseInt($parent.css('padding-top'));
+        const paddingLeft = parseInt($parent.css('padding-left'));
+        const paddingRight = parseInt($parent.css('padding-right'));
+
+        const center = (paddingLeft + paddingRight) + ((($parent[0].clientWidth - (paddingLeft + paddingRight)) / 2) - this.$el[0].offsetWidth / 2);
+        const topPosition = $parent[0].offsetTop;
+        let top;
+        if (this.viewPortState === 'up') {
+            top = topPosition + this.$el.innerHeight() + paddingTop;
+            this.$el.css({top: top, left: center});
+        } else if (this.viewPortState === 'down') {
+            top = topPosition + $parent.innerHeight() - this.$el.innerHeight() * 2;
+            this.$el.css({top: top, left: center});
+        } else {
+            this.$el.position({
+                my: this._get_spaced_inverted_position(appendAt),
+                at: appendAt,
+                of: this.$anchor,
+                collision: "none",
+            });
+        }
 
         // Reverse overlay if direction is right to left
         var positionRight = _t.database.parameters.direction === 'rtl' ? "right" : "left";
         var positionLeft = _t.database.parameters.direction === 'rtl' ? "left" : "right";
-        var offset = this.$el.offset();
+
+        // get the offset position of this.$el
+        // Couldn't use offset() or position() because the first one take the gap between the element and the document
+        // and the second one seems to take the gap between the element and the parent or the window (If the top of the parent is out of bound)
+        const offset = {top: this.$el[0].offsetTop, left: this.$el[0].offsetLeft};
         this.$tooltip_overlay.css({
```

Comment:
This is probably because you used position: fixed so it is probably not necessary.